dotnet/src/Connectors/Connectors.AI.Oobabooga/TextCompletion/OoobaboogaTextCompletion.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Filename doesn't match class name. I think there is extra "o" at the beginning of filename 😃

Reply:
See [#cf742c4f2d9df1fa48f1410e34670d625e66995f](https://github.com/microsoft/semantic-kernel/pull/1357/commits/cf742c4f2d9df1fa48f1410e34670d625e66995f)